internal/docstore/docstore.go

```diff
@@ -100,20 +109,29 @@ func (l *ActionList) Create(doc Document) *ActionList {
 // The key fields must be set.
 // The document must already exist; an error for which gcerrors.Code returns NotFound
 // is returned if it does not.
+// If the document has a non-nil RevisionField, then an error for which gcerrors.Code
```

Comment:
In general, I think these comments should be left off from individual APIs, maybe it is enough to add those to the package doc, WDYT?

Reply:
Done.